lighthouse-core/gather/gatherers/source-maps.js

```diff
@@ -130,6 +140,7 @@ class SourceMaps extends Gatherer {
 
     driver.off('Debugger.scriptParsed', this.onScriptParsed);
     await driver.sendCommand('Debugger.disable');
+    await driver.sendCommand('Network.enable');
```

Comment:
should probably move any network enable/disable to Fetcher, once your new fetch code lives there.